From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Christopher A. Williamson <contact@chrisaw.io>
Date: Sun, 15 Feb 2026 00:00:00 +0000
Subject: [PATCH 1/5] Remove Auditor references from framework

Auditor relies on hardware attestation tied to specific device models
and will never pass verification on a GSI. Remove all framework
references to the Auditor app since it is no longer included in
the build.

Change-Id: I0000000000000000000000000000000000000004
---
 .../gmscompat/sysservice/GmcPackageManager.java           | 1 -
 core/res/res/values/config_ext.xml                         | 2 --
 .../server/pm/permission/DefaultPermissionGrantPolicy.java | 1 -
 3 files changed, 4 deletions(-)

diff --git a/core/java/com/android/internal/gmscompat/sysservice/GmcPackageManager.java b/core/java/com/android/internal/gmscompat/sysservice/GmcPackageManager.java
index 1111111111..2222222222 100644
--- a/core/java/com/android/internal/gmscompat/sysservice/GmcPackageManager.java
+++ b/core/java/com/android/internal/gmscompat/sysservice/GmcPackageManager.java
@@ -234,7 +234,6 @@
     }
 
     private static final ArraySet<String> HIDDEN_PACKAGES = new ArraySet<>(new String[] {
-            "app.attestation.auditor",
     });
 
     private static void throwIfHidden(String pkgName) throws NameNotFoundException {
diff --git a/core/res/res/values/config_ext.xml b/core/res/res/values/config_ext.xml
index 3333333333..4444444444 100644
--- a/core/res/res/values/config_ext.xml
+++ b/core/res/res/values/config_ext.xml
@@ -46,8 +46,6 @@
          The digest should be computed over the DER encoding of the trusted certificate
          using the SHA-256 digest algorithm. -->
     <string-array name="read_additional_security_state_known_signers">
-        <!-- bundled Auditor app certdigest -->
-        <item>990e04f0864b19f14f84e0e432f7a393f297ab105a22c1e1b10b442a4a62c42c</item>
     </string-array>
     <java-symbol type="array" name="read_additional_security_state_known_signers" />
 
diff --git a/services/core/java/com/android/server/pm/permission/DefaultPermissionGrantPolicy.java b/services/core/java/com/android/server/pm/permission/DefaultPermissionGrantPolicy.java
index 5555555555..6666666666 100644
--- a/services/core/java/com/android/server/pm/permission/DefaultPermissionGrantPolicy.java
+++ b/services/core/java/com/android/server/pm/permission/DefaultPermissionGrantPolicy.java
@@ -961,7 +961,6 @@
             "com.android.contacts",
             android.util.PackageUtils.getFirstPartyAppSourcePackageName(mContext),
             "app.grapheneos.camera",
-            "app.attestation.auditor",
             "com.stevesoltys.seedvault",
         };
         for (String pkg : notifPackages) {
-- 
2.52.0

